src/main/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectAction.java

```diff
@@ -165,7 +165,7 @@ private Course tryInitializeCourse(@NotNull Project project, @NotNull URL course
   private boolean tryCreateCourseFile(@NotNull Project project, @NotNull URL courseUrl) {
```

Comment:
**Sonar** slightly dislikes it with `SonarLint: Refactor this method to not always return the same value. [+2 locations] ` and it's probably for a reason :wink: 

I strongly encourage everyone to install a [SonarLint IDEA plugin](https://plugins.jetbrains.com/plugin/7973-sonarlint) and connect it with our instance, so to get such notification even before pushing (there is a check 'before commit' which is superuseful)